client/index.js

```diff
@@ -122,6 +123,14 @@ const onSocketMsg = {
       }
     }
   },
+  progress(progress) {
+    if (typeof document !== 'undefined') {
+      useProgress = progress;
+    }
+  },
+  'progress-update': function progressUpdate(data) {
+    if (useProgress) log.info(`[WDS] ${data.percent}% - ${data.msg}.`);
```

Comment:
Isn't this super spammy?

Reply:
given that it's an opt-in and the browser console doesn't have control characters, I don't believe it is.